src/components/IconBadge/IconBadge.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import classNames from 'classnames'
+import { Props } from './IconBadge.types'
+import './IconBadge.css'
+
+const IconBadge = ({ icon, text, onClick, className, children }: Props) => {
```

Comment:
It might be interesting to have the IconBadge with stories, what do you think?

Reply:
agreed, done!